submissions/dmytro-botnarenko/html-css-popup/style.css

```diff
@@ -0,0 +1,153 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+  background-color: white;
+  font-family: Tahoma, Helvetica, Arial, sans-serif;
+  color: #fff;
+  font-size: calc(12px + 0.390625vw);
```

Comment:
why such a strange number?

Reply:
I used [this article](https://matthewjamestaylor.com/responsive-font-size#:~:text=Fluid%20Responsive%20Font%20Size) to implement responsive `font-size`. It is possible that this is a very subjective opinion. Now I've changed it to `font-size: calc(15px + 0.39vw)`. Is this ok?